metrics/ci_integrations/lib/source/jenkins/adapter/jenkins_source_client_adapter.dart

```diff
@@ -141,7 +141,7 @@ class JenkinsSourceClientAdapter implements SourceClient {
     final buildDataFutures = builds.where((build) {
       return _checkBuildFinishedAndInRange(build, startAfterBuildNumber);
     }).map((build) async {
```

Comment:
```suggestion
    }).map((build) => _mapJenkinsBuild(jobName, build));
```
Would it be better? If formatting is bad, then just get rid of the async modifier.